src/main/java/com/grim3212/mc/pack/industry/block/BlockUraniumOre.java

```diff
@@ -22,46 +24,77 @@ public BlockUraniumOre() {
-				damage--;
+				//damage--;
+				damage = damage - damageFraction;
+				if (damage < 0) damage = 0;
 			}
 			if (((EntityPlayer) entityIn).inventory.armorInventory[2] != null && ((EntityPlayer) entityIn).inventory.armorInventory[2].getItem() == IndustryItems.anti_radiation_chest) {
-				damage--;
+				//damage--;
+				damage = damage - damageFraction;
+				if (damage < 0) damage = 0;
 			}
 			if (((EntityPlayer) entityIn).inventory.armorInventory[3] != null && ((EntityPlayer) entityIn).inventory.armorInventory[3].getItem() == IndustryItems.anti_radiation_helmet) {
-				damage--;
+				//damage--;
+				damage = damage - damageFraction;
+				if (damage < 0) damage = 0;
 			}
 			if (damage != 0) {
 				entityIn.attackEntityFrom(DamageSource.magic, damage);
 			}
 		} else {
 			if (damage != 0) {
+				worldIn.spawnParticle(EnumParticleTypes.REDSTONE, pos.getX() + 0.5D, pos.getY() + 1.0D, pos.getZ() + 0.5D, 0.0D, 4.0D, 0.0D, new int[0]);
+				worldIn.spawnParticle(EnumParticleTypes.REDSTONE, pos.getX() + 0.5D, pos.getY() + 1.25D, pos.getZ() + 0.5D, 0.0D, 4.0D, 0.0D, new int[0]);
+				worldIn.spawnParticle(EnumParticleTypes.REDSTONE, pos.getX() + 0.5D, pos.getY() + 1.5D, pos.getZ() + 0.5D, 0.0D, 4.0D, 0.0D, new int[0]);
+				worldIn.spawnParticle(EnumParticleTypes.REDSTONE, pos.getX() + 0.5D, pos.getY() + 1.75D, pos.getZ() + 0.5D, 0.0D, 4.0D, 0.0D, new int[0]);
+				worldIn.spawnParticle(EnumParticleTypes.REDSTONE, pos.getX() + 0.5D, pos.getY() + 2.0D, pos.getZ() + 0.5D, 0.0D, 4.0D, 0.0D, new int[0]);
+
 				entityIn.attackEntityFrom(DamageSource.magic, damage);
 			}
 		}
 	}
 
 	@Override
 	public void onBlockClicked(World worldIn, BlockPos pos, EntityPlayer playerIn) {
-		int damage = 4;
+
+		float damage = IndustryConfig.uraniumDamage;
+		float damageFraction = damage/4;
 		if (playerIn.inventory.armorInventory[3] != null && playerIn.inventory.armorInventory[3].getItem() == IndustryItems.anti_radiation_helmet) {
-			damage--;
+			//damage--;
+			damage = damage - damageFraction;
+			if (damage < 0) damage = 0;
 		}
 		if (playerIn.inventory.armorInventory[2] != null && playerIn.inventory.armorInventory[2].getItem() == IndustryItems.anti_radiation_chest) {
-			damage--;
+			//damage--;
+			damage = damage - damageFraction;
+			if (damage < 0) damage = 0;
 		}
 		if (playerIn.inventory.armorInventory[1] != null && playerIn.inventory.armorInventory[1].getItem() == IndustryItems.anti_radiation_legs) {
-			damage--;
+			//damage--;
+			damage = damage - damageFraction;
+			if (damage < 0) damage = 0;
 		}
 		if (playerIn.inventory.armorInventory[0] != null && playerIn.inventory.armorInventory[0].getItem() == IndustryItems.anti_radiation_boots) {
-			damage--;
+			//damage--;
+			damage = damage - damageFraction;
+			if (damage < 0) damage = 0;
 		}
 		if (damage != 0) {
+			worldIn.spawnParticle(EnumParticleTypes.REDSTONE, pos.getX() + 0.5D, pos.getY() + 1.0D, pos.getZ() + 0.5D, 0.0D, 4.0D, 0.0D, new int[0]);
```

Comment:
If you think so, yes. You have the final say always. One thing I do want to point out, if remembering correctly, in my 1.8.9 world with Grim I was getting damage by just walking on the Uranium blocks. That actually sucked near bedrock and lava. 1.10.2 doesn't do this. There is some playerCollide code I saw. If it is meant to be this way, it doesn't work on 1.10.2 and as I said for that I am glad.

You might also want to adjust the spawn rate a bit. There is a lot of Uranium at the lower levels.